index.js

```diff
@@ -68,6 +68,22 @@ function formatWithCursor(text, opts, addAlignmentSize) {
   }
 
   text = stripBom(text);
+
+  if (
+    opts.insertPragma &&
+    !hasPragma(text) &&
+    opts.rangeStart === 0 &&
+    opts.rangeEnd === Infinity
+  ) {
+    const parsedDocblock = docblock.parseWithComments(docblock.extract(text));
+    const pragmas = Object.assign({}, { format: "" }, parsedDocblock.pragmas);
```

Comment:
You don't need the first argument to be an empty object literal when the second one is a fresh object too.

Reply:
good point. updating